src/Mapper.cpp

```diff
@@ -1520,21 +1531,25 @@ int Mapper::incrementalMap(CGRA* t_cgra, DFG* t_dfg, int t_II){
       for (auto fu : placementRecommList) {
         map<CGRANode*, int>* path = calculateCost(t_cgra, t_dfg, t_II, *dfgNode, fu);
         if (path == NULL) {
+          // Switches to next tile in placementRecommList.
           cout<<"[DEBUG] no available path for DFG node "<<(*dfgNode)->getID()<<" on CGRA node "<<fu->getID()<<" within II "<<t_II<<endl;
-          continue; // Switches to next tile in placementRecommList
+          continue; 
         }
         else {
           if (schedule(t_cgra, t_dfg, t_II, *dfgNode, path, false)) {
+            // Current DFGNode finishes mapping, move to next DFGNode.
             dfgNodeMapFailed = false;
-            break; // Current DFGNode finishes mapping, move to next DFGNode
+            break; 
           }
           else {
+            // Switches to next tile in placementRecommList.
             cout<<"[DEBUG] no available path to schedule DFG node "<<(*dfgNode)->getID()<<" on CGRA node "<<fu->getID()<<" within II "<<t_II<<endl;
-            continue; // Switches to next tile in placementRecommList
+            continue; 
           }
         }
       }
-      if (dfgNodeMapFailed) break; // Current DFGNode fails mapping, increase II and restart
+      // Current DFGNode fails mapping, increase II and restart.
```

Comment:
`// Increases II and restart if current DFGNode fails the mapping.`

Reply:
Has been corrected